main/inputoutput/io.py

```diff
@@ -1,2 +1,9 @@
+from zipfile import ZipFile
 def inc(num):
-    return num+2
\ No newline at end of file
+    return num+2
+
+def unzip_file(fromPath,toPath):
+    file=ZipFile(fromPath,'r')
```

Comment:
In Pycharm, if you see some highlighting with underlines, it usually means that pep-8 conventions are not being followed. 

For example, there are 4 places in the below code with underlines
<img width="286" alt="image" src="https://user-images.githubusercontent.com/3809547/54459035-68f61880-475d-11e9-815c-4efbc4a5996a.png">

You can fix those in pycharm directly, as you will get more information when you point your mouse on the underlined item

<img width="282" alt="image" src="https://user-images.githubusercontent.com/3809547/54459130-a9559680-475d-11e9-8302-da0966f53ea3.png">


If you want to check all errors in all files, then you can run pycodestyle from the terminal/command prompt


<img width="615" alt="image" src="https://user-images.githubusercontent.com/3809547/54459198-d6a24480-475d-11e9-99b1-55ab02de40a5.png">